specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimbackends.json

```diff
@@ -307,6 +307,57 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/backends/{backendid}/reconnect": {
+      "post": {
+        "tags": [
+          "Backends"
+        ],
+        "operationId": "Backend_Reconnect",
+        "description": "Initiates .",
```

Comment:
>Initiates [](start = 24, length = 9)

improve description, this is what the customer sees when he will land on this api. Questions that are good to be answered
1. Why would you want to set this property
2. How it behaves
3. Is there blog link you can link, where customer can read more.